crates/fuel-streams-core/src/stream.rs

```diff
@@ -90,6 +87,7 @@ impl<S: Streamable> Stream<S> {
             .get_or_create_kv_store(kv::Config {
                 bucket: bucket_name.to_owned(),
                 storage: stream::StorageType::File,
+                history: 1,
```

Comment:
This property is used to avoid a store entry to have more than one revision at its history

Reply:
ok, got it.